ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/provision/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision-chain.yaml

```diff
@@ -0,0 +1,100 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+  steps: 
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-gcp-user-labels-tags
+    - ref: ipi-conf-credentials-mode
+    - ref: gcp-provision-user-tags-serviceaccount
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
+    - ref: cucushift-installer-check-gcp-user-labels
```

Comment:
If I remember right, because we do not find out gcloud command to check tag setting, so we do not cover the tag check yet, right?

Reply:
Yes, automation doesn't do the tag check.